### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by scarabus**

### CODE REVIEW

**Comment by scarabus** [comment]
Don't merge this yet.

**Comment by scarabus** [comment]
Don't merge this yet.

**Comment by jonathan-major** [comment]
is the pr ready?

File: Apps/RaptureIntTests/build.gradle [hunk 1]
```diff
@@ -77,3 +73,18 @@ task docTests (type: Test) {
...
+    commandLine nosetestsCmd, 'src/test/python/searchTests.py'
+}
+
+task pythonTests(type: Test, dependsOn: pythonSearchTests) {
+    description 'Run Python tests'
+    println 'Running Python tests'
+}
+
+// This doesn't seem to work
+pythonTests.mustRunAfter nightlyTests
```
Comment (jonathan-major) [inline Apps/RaptureIntTests/build.gradle]: then why leave it in?

**Comment by jeremy-flieder** [comment]
There is still a bit of work to do on the reflex entitlement test but will merge to at least get up to date w/ everything else

File: Apps/RaptureIntTests/build.gradle (hunk 1 above)
Comment (scarabus) [inline Apps/RaptureIntTests/build.gradle]: As a reminder that it doesn't work, in the hope that either I or someone else will figure out what the problem is. It doesn't cause a problem, but it's supposed to have the Python tests run after the Java tests - and that part isn't happening.

**Pull request merged**